test/test_tensor.h

```diff
@@ -927,11 +927,11 @@ TEST(tensor, nd1) {
 }
 
 TEST(tensor, print) {
-    Tensor<float_t, 4> t({3,2,2,2});
+    Tensor<float_t, 5> t({3,2,2,2,2});
```

Comment:
@Randl Thank you for the explanation. For debugging, writing a Pretty-Printer with Python code for gdb and Natvis XML file for Visual Studio would be beneficial.